site/quickstarts/index.html

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
Did we review/decide this url structure?

`/quickstarts` vs `/docs/quickstarts`?

I don't remember.

Reply:
This lives in docs. So` /docs/quickstarts`